File: src/main/java/com/micabytes/ink/InkParseException.java
package com.micabytes.ink;

@SuppressWarnings("unused")
public class InkParseException extends Exception {

  public InkParseException(String message) {
    super(message);
  }

  public InkParseException(Throwable throwable) {
    super(throwable);
  }

  public InkParseException(String message, Throwable throwable) {
    super(message, throwable);
  }

}

File: src/main/kotlin/com/micabytes/ink/InkParseException.kt
package com.micabytes.ink

@SuppressWarnings("unused")
class InkParseException : Exception {

    constructor(message: String) : super(message) {
    }

    constructor(throwable: Throwable) : super(throwable) {
    }

    constructor(message: String, throwable: Throwable) : super(message, throwable) {
    }

}
